Sources/Crypto.m

```diff
@@ -168,12 +168,24 @@ + (nullable NSData *)bech32Decode:(nonnull NSString *)string hrp:(NSString * _Nu
     uint8_t data[82];
     char hrpBuf[84];
     size_t dataLen;
+
     if (1 != bech32_decode(hrpBuf, data, &dataLen, string.UTF8String)) {
         return nil;
     };
     if (hrp) {
         *hrp = [NSString stringWithUTF8String:hrpBuf];
+        NSArray<NSString*>* bitcoinsHRP = @[@"bc",@"tb"];
```

Comment:
This part is tricky, since this is the general decoding method, what if `"ltc"` for Litecoin? It's better to move this validation to `BitcoinSegwitAddress` or somewhere